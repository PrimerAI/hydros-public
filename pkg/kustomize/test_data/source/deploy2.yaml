apiVersion: apps/v1
kind: Deployment
metadata:
  labels:
    app: deploy2
  name: deploy2
  namespace: cert-manager
spec:
  replicas: 1
  selector:
    matchLabels:
      app: cert-manager
    metadata:
      labels:
        app: cert-manager
  template:
    spec:
      containers:
        - image: some/repo/gateway:latest
          name: gateway
