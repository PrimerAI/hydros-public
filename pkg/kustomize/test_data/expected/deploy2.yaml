apiVersion: apps/v1
kind: Deployment
metadata:
  name: deploy2
  namespace: cert-manager
  labels:
    app: deploy2
spec:
  replicas: 1
  selector:
    metadata:
      labels:
        app: cert-manager
    matchLabels:
      app: cert-manager
  template:
    spec:
      containers:
        - name: gateway
          image: some/repo/gateway:latest
